yt_dlp/extractor/vrt.py

```diff
@@ -1,42 +1,48 @@
 import json
 
-from time import (strftime, gmtime)
 from .gigya import GigyaBaseIE
-from ..compat import compat_HTTPError
 from ..utils import (
     parse_iso8601,
     ExtractorError,
-    int_or_none,
+    clean_html,
     float_or_none,
+    int_or_none,
+    join_nonempty,
+    make_archive_id,
     str_or_none,
+    traverse_obj,
     url_or_none,
-    urlencode_postdata
+    urlencode_postdata,
+    urllib,
 )
 
 
 class VRTIE(GigyaBaseIE):
     IE_DESC = 'VRT'
     _VALID_URL = r'https?://(?:www\.)?vrt\.be/vrtnu/a-z/(?:[^/]+/){2}(?P<id>[^/?#&]+)'
     _TESTS = [{
-        'url': 'https://www.vrt.be/vrtnu/a-z/de-ideale-wereld/2023-vj/de-ideale-wereld-d20230116/',
+        'url': 'https://www.vrt.be/vrtnu/a-z/buurman--wat-doet-u-nu-/6/buurman--wat-doet-u-nu--s6-trailer/',
         'info_dict': {
-            'id': 'pbs-pub-855b00a8-6ce2-4032-ac4f-1fcf3ae78524$vid-d2243aa1-ec46-4e34-a55b-92568459906f',
-            'title': 'Tom Waes',
-            'thumbnail': 'https://images.vrt.be/orig/2023/01/10/1bb39cb3-9115-11ed-b07d-02b7b76bf47f.jpg',
-            'ext': 'mp4',
-            'duration': 1939.0,
-            'release_date': '20230116',
-            'description': 'Satirisch actualiteitenmagazine met Ella Leyers. Tom Waes is te gast.',
-            'display_id': 'de-ideale-wereld-d20230116',
-            'episode_number': 1,
-            'timestamp': 1673905125,
-            'series': 'De ideale wereld',
-            'upload_date': '20230116',
+            'title': "Trailer seizoen 6 'Buurman, wat doet u nu?'",
+            'description': "Vanaf donderdag 26 mei trekt Cath Luyten naar het buitenland voor de zesde reeks van 'Buurman, wat doet u nu?'. Dit keer mag ze gaan logeren bij boeiende, bekende landgenoten die hun Belgische bestaan – tijdelijk of definitief - omruilden voor een nieuw leven buiten onze landsgrenzen. \n'Buurman, wat doet u nu?' staat dit zesde seizoen meer dan ooit voor \"hoe gaat het nog met u?\" Een vraag waarmee Cath Luyten internationaal mag aankloppen bij Sam Bettens in Palm Desert (VS), VRT-coryfee Jessie De Caluwe in Marokko, voormalig zanger en interviewer Ivan Heylen op de Canarische Eilanden, topkok Piet Huysentruyt in Zuid-Afrika en voormalig politicus Johan Van Hecke en zijn echtgenote, journaliste Els De Temmerman, in Oeganda. Eén voor één ontvangen ze Cath om haar twee dagen lang onder te dompelen in hun buitenlandse bestaan. Zo krijgt Cath een unieke inkijk in het internationale leven van haar gasten. Samen met hen beschouwt ze hun verleden, heden en toekomst en ontdekt ze waarom ze hun hart verloren in het buitenland.",
```

Comment:
use md5 for long descriptions

change the value to `md5:TODO` and run the test again, it will fail and output hash value